README.md

```diff
@@ -195,53 +195,40 @@ An [Effect] are the instruction to the [Executor] to run set of [Task]s concurre
 
 ```ipldsch
 type Task struct {
-  on      URI
-  call    Ability
-  input   {String : Any}
-
-  nnc     string
+  uri    URI
+  call   Ability
+  input  {String : Any}
+  nnc    String
 }
 
 type URI string
 type Ability string
 
-type Authorization struct {
-  # Authorization is denoted by the set of links been authorized
-  scope   [&Any]
-  # Scope signed by the invoker
-  s       VarSig
-}
-
-type Invocation struct {
-  v       SemVer
-
+type Context struct {
   run     &Task
+  meta    {String : Any}
+  prf     [&UCAN]
 
   # Receipt of the invocation that caused this invocation
   cause   optional &Invocation
+}
 
-  # Task authorization.
-  auth    &Authorization
-
-  meta    {String : Any}
-
-  prf     [&UCAN]
+type Invocation struct {
+  ctx     Context
+  sig     Varsig
```

Comment:
I changed this from `Authorization` to a varsig, since I'm like 99% certain that we can define a multiple signature scheme as a varsig. That said, let know if this is too radical and you want `Authorization` back. It does work for the IPVM use case, this just seemed like an opportunity to make this consistent.

Reply:
I’m not sure varsig is going to address it because set of block cids need to be carried somewhere otherwise how are you going to derive them from just a payload ?

you could encode cid set into varsig but I’m not convinced it’s better than actual structure in the data model.

I think I wouldn’t change it until there’s a spec for such varsig variant